_presenters/tim-schilling.md

```diff
@@ -1,4 +1,6 @@
 ---
+difficulty: Beginner
```

Comment:
```suggestion

```

The difficulty goes on your [tutorial page](https://github.com/djangocon/2022.djangocon.us/blob/main/_schedule/tutorials/2022-10-16-13-30-t0-it-doesnt-work-a-djangonauts-debugging.md), not your speaker bio.

Reply:
I'll take care of that in a follow-up PR. Thanks!